respy/interpolate.py

```diff
@@ -132,6 +117,38 @@ def _kw_94_interpolation(
     return period_expected_value_functions
 
 
+def _get_seeds_for_interpolation(state_space, options):
+    if hasattr(state_space, "sub_state_spaces"):
+        seed = {
+            dense_idx: next(options["solution_seed_iteration"])
+            for dense_idx in state_space.sub_state_spaces
+        }
+    else:
+        seed = next(options["solution_seed_iteration"])
+
+    return seed
+
+
+def _split_interpolation_points_evenly(state_space, options):
+    """Split the number of interpolated states evenly across dense dimensions."""
```

Comment:
Why do we only split evenly across dense dimensions again? 
Wouldnt that also be useful for e.g. experiences or the like?

Reply:
We parallelize the computation of the lhs variable over dense dimensions. Say the user wants to base the interpolation on 500 states, we take an almost equal number of states from each dense dimension. Within each dense dimension we have the same combination of experiences, lagged choices etc.

Regarding our current work-in-progress on the state space, this function gets more complicated as all `SubStateSpace`s in one period do not share the same number of states. We would probably number all states in one period, take 500 random states and pass the information back to `SubStateSpace`s so that these 500 states are not interpolated.

Does it help? :)